### ISSUE TITLE
Installation on Jetson boards

### ISSUE THREAD

**Issue by miziworld** [espnet/espnet#4385]
Hello, I am trying to run espnet on various embedded boards.
In the case of Raspberry Pi, it runs smoothly, but in the case of the board of the jetson target, it is very difficult to match the version and dependencies of pytorch and related packages.
Also, in the case of jetpack, python3.6 is installed by default, but espnet is supported from 3.7 or later, so it is not easy to install pytorch corresponding to python3.7 in the jetson environment.
Is it absolutely impossible to run espnet with python3.6?
Also, if you have experience installing espnet in the jetson environment, please share the development environment version.

**Comment by kamo-naoyuki** [comment]
I think it works for almost all tasks, but it depens on your purpose.
It's better to try python3.6  and, if some problems happen, please ask it again.

**Comment by miziworld** [comment]
I installed on Jetson nano board, and I think every installation is successed but the decoded results are weired.
The trained model works well with my PC environment with high accuracy, and I use the same model on jetson board.
What's the difference? and how to solve it? 
My environment :
- python 3.7.9
- torch : 1.10.0 + cuda 10.2
- espnet2 / asr model
- decoding with device=cuda
- Done with locale setup
![image](https://user-images.githubusercontent.com/17849068/169827753-4491b436-57b6-46af-bf3c-1f8f685c73a0.png)

**Comment by kamo-naoyuki** [comment]
Finally, did you use python3.7? I know some differences between python3.7 and python3.6, but I don't know about jetson, sorry. 

Does it work for CPU mode (does it exist?)?

**Comment by miziworld** [comment]
I succesfully reinstalled pytorch on Jetson environment and the problem disappeared. I used the wheel provided NVIDIA, and the version is pytorch 1.8.0 and python 3.8 It works both CPU  and GPU mode.
